src/APL-1.0.xml

```diff
@@ -0,0 +1,970 @@
+<SPDX name="Adaptive Public License 1.0" identifier="APL-1.0" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/APL-1.0</url>
+  </urls>
+  <license>
+    <title>
+      <p>ADAPTIVE PUBLIC LICENSE 
+        <br/>Version 1.0 
+      </p>
+    </title>
+    <body>
+      <p>THE LICENSED WORK IS PROVIDED UNDER THE TERMS OF THIS ADAPTIVE PUBLIC LICENSE ("LICENSE"). ANY USE,
+         REPRODUCTION OR DISTRIBUTION OF THE LICENSED WORK CONSTITUTES RECIPIENT'S ACCEPTANCE OF THIS LICENSE
+         AND ITS TERMS, WHETHER OR NOT SUCH RECIPIENT READS THE TERMS OF THIS LICENSE. "LICENSED WORK" AND
+         "RECIPIENT" ARE DEFINED BELOW.</p>
+      <p>IMPORTANT NOTE: This License is "adaptive", and the generic version or another version of an Adaptive
+         Public License should not be relied upon to determine your rights and obligations under this License.
+         You must read the specific Adaptive Public License that you receive with the Licensed Work, as certain
+         terms are defined at the outset by the Initial Contributor.</p>
+      <p>See Section 2.2 below, Exhibit A attached, and any Suppfile.txt accompanying this License to determine
+         the specific adaptive features applicable to this License. For example, without limiting the
+         foregoing, (a) for selected choice of law and jurisdiction see Part 3 of Exhibit A; (b) for the
+         selected definition of Third Party see Part 4 of Exhibit A; and (c) for selected patent licensing
+         terms (if any) see Section 2.2 below and Part 6 of Exhibit A.</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>DEFINITIONS.</p>
+        </li>
+        <li>
```

Comment:
Consider starting a new \<list> here, since strictly 1.1, 1.2 etc are children of 1.